spec/classes/clamav_spec.rb

```diff
@@ -5,6 +5,11 @@
     context "on #{os}" do
       let(:facts) do
         facts.merge(:environment => 'test')
+        if os =~ /-7-/
```

Comment:
Probably use the facts themselves to test...
https://github.com/edestecd/puppet-mariadb/blob/master/spec/classes/mariadb_repo_spec.rb#L12

Reply:
Sure thing; done.